Resuelve tarea 2

NicolasFernandez-Kinar32 opened the pull request.

NicolasFernandez-Kinar32 committed:
Resuelve tarea 2

fsodano commented on tarea2.html:
```diff
@@ -0,0 +1,32 @@
+
+    <main>
+        <div id="saludo">Hola</div>  
+        <div class="wrapper" id="caja-respuesta"> 
+            <div class="box" id="respuesta-nombre"></div>
+        </div>
```
ojo con el spanglish

fsodano commented on tarea2.js:
```diff
@@ -0,0 +1,17 @@
+    document.querySelector("#respuesta-nombre").innerText = saludoCompleto;
+    return false;
+}
+/*
+document.querySelector("#boton-ingresar").onclick = function (){
+    const $nombreUsuario = document.querySelector("#nombre-usuario").value;
+    document.querySelector("h1").innerText = `Bienvenido ${$nombreUsuario}`;
+    return false;
+}
+*/
```
Nunca se commitea código muerto, así que esto borralo! Si este código existía en otro commit, podés ir a revisar el historial y verlo ahí.

NicolasFernandez-Kinar32 committed:
Corrige errores finales en tarea

Si bien el .js era perfectamente funcional, se cambia el modo de resolución como sugerencia del revisor, además de un pequeño problema con los idiomas en el .html
